README.md

```diff
@@ -10,51 +10,142 @@
 
 [Contentful](https://www.contentful.com) provides a content infrastructure for digital teams to power content in websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management and delivery APIs, and a customizable web app that enable developers and content creators to ship digital products faster.
 
-This Command Line Tool (CLI) helps you to import files generated by [contentful-export](https://github.com/contentful/contentful-export) to a destination space.
+This library helps you to import files generated by [contentful-export](https://github.com/contentful/contentful-export) to a destination space.
 
-## Installation
+## :exclamation: Usage as CLI
+We moved the CLI version of this tool into our [Contentful CLI](https://github.com/contentful/contentful-cli). This allows our users to use and install only one single CLI tool to get the full Contentful experience.
```

Comment:
same comment as my review of contentful-export. I want some more prominence of this entire sentence to make it stand out to the eye more. i suspect most usage will be via contentful-cli, so we gotta make sure people find that ;-)